### DISCUSSION TITLE
showing list items from those who i have unfollowed

### DISCUSSION THREAD

**Discussion by goyalyashpal** [community/community#46846] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Still showing lists from those who i have unfollowed - even after force refreshing, and opening on another device etc...

[image](https://user-images.githubusercontent.com/19423063/217816544-7eef0a2f-7816-47b2-a1e5-7cc689a50e65.png)

**Comment by goyalyashpal** [comment, marked as answer]
I think that this page called "dashboard" does NOT immediately fetch and show the updates from all of ur current follow subscriptions - thats what other SNS do.

Rather, gh's dashboard seems more of a static page _to which_ items are pushed when an event occurs, so, like RSS feed of sorts I guess.

So, the old feed items stay there.

If thats the case, then having this in documentation etc will help.

**Reply by dzholkov** [reply]
У меня вообще сейчас ничего не грузится в браузере

**Reply by goyalyashpal** [reply]
> _У меня вообще сейчас ничего не грузится в браузере_

translation?
